test/test_linear_kinematic.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from src.linear_kinematic import LinearKinematic
+from src.vec2d import Vec2d
+from src.common_constants import PhysicalBoundaries
+
+ABS_TOLERANCE = 1e-5
+
+
+class TestLinearKinematic:
+
+    def test_set_mass(self):
+        obj = LinearKinematic(mass=5.0)
+        assert obj.mass == 5.0
+        with pytest.raises(ValueError):
```

Comment:
Versteh ich noch nicht ganz, sieht aber interessant aus

Reply:
der code in dem `with.pytest.raises(ValueError):` abschnitt muss einen ValueError werfen, nur dann ist erfolgreich. das stellt sicher, dass die funktion sich über negative massen beschwert, falls sie denn welche bekommt